jovian/cli.py

```diff
@@ -1,10 +1,11 @@
 import argparse
-import webbrowser
+# import webbrowser
```

Comment:
Don't uncomment code, remove it if it's not used. Comments should only be used for documentation.